### DISCUSSION TITLE
funds missing in "unconfirmed" balance from closed channel

### DISCUSSION THREAD

**Discussion by ashjkl159** [lightningnetwork/lnd#8012] [Troubleshooting] [answered]
### Background
First I tried to openchannel with low onchain fee. After more then 2 weeks of not-confirming, I did `lncli bump fee channelpoint:outputindex`. Channel got finally confirmed but was inactive probably because of longer then 2week of not-confirming. Therefore, i tried to force close it. After force closing it (via LiT terminal), it shows in `lncli pendingchannels` but not as "pending_force_closing_channels" but in "waiting_close_channels" with "ChanStatusBorked|ChanStatusCommitBroadcasted|ChanStatusLocalCloseInitiator", when i do `lncli closechannel --funding_txid --output_index --force` to force close it again, it throws error ... "channel is already in the process of being force closed". I can see closing_txid in mempool.space, but `lncli walletbalance` cannot see those funds "unconfirmed"

### Your environment

* version of `lnd` lnd 16.4
* which operating system (`uname -a` on *Nix) ubuntu server 22  5.15.0-83
* version of `btcd`, `bitcoind`, or other backend - bitcoind
* any other relevant environment details

### Steps to reproduce

lncli openchannel
lncli wallet bumpfee --sat_per_byte xx outputpoint:index
force closing channel via LiT terminal (v10.5)

### Expected behaviour

I was expecting to have this channel force closed, in `lncli pendingchannels` as "pending_force_closing_channels"

### Actual behaviour

in `lncli pendingchannels` i have this channel under "waiting_close_channels" with "ChanStatusBorked|ChanStatusCommitBroadcasted|ChanStatusLocalCloseInitiator"

**Comment by bitromortac** [comment]
Was the force close transaction confirmed? If yes then the next step is that you have to wait for the CSV timelock to expire in order to get the channel fully resolved, which could take some days.

**Comment by ashjkl159** [comment]
could it be that when i was bumping fee in openning channel (lncli wallet bumpfee --sat_per_byte xx outputpoint:index) .... i accidentaly put --sat_per_byte (not --sat_per_vbyte) so channel got openned with closing non segwit address and that is why i cannot see it in my onchain segwit wallet?

**Comment by bitromortac** [comment]
That shouldn't have to do with it. It would be helpful if you could share the txid or give more info about the confirmation status of the force close.

**Comment by saubyk** [comment]
Moving this issue to a discussion thread, as there is nothing for us to fix here

**Comment by ashjkl159** [comment]
closing_txid 3e8e8747775d827899ec12a6b2fec09c3b678847cea7c93e5ceb7d1b2065b808

i can see txn in mempool with fee 11.5 (i would like to bump fee) but I cannot find anywhere any address in `lncli listunspent` or i do not see those funds coming in `lncli walletbalance` 

this is from `lncli listchaintxns`
"tx_hash": "3e8e8747775d827899ec12a6b2fec09c3b678847cea7c93e5ceb7d1b2065b808",
            "amount": "0",
            "num_confirmations": 0,
            "block_hash": "",
            "block_height": 0,
            "time_stamp": "1694701744",
            "total_fees": "0",
            "dest_addresses": [
                "bc1qfu02sn4hk5rvljq55psrsyy4uufjkk43d7lgf3dgkytu8wl24awspqmtze"
            ],
            "output_details": [
                {
                    "output_type": "SCRIPT_TYPE_WITNESS_V0_SCRIPT_HASH",
                    "address": "bc1qfu02sn4hk5rvljq55psrsyy4uufjkk43d7lgf3dgkytu8wl24awspqmtze",
                    "pk_script": "00204f1ea84eb7b506cfc814a060381095e7132b5ab16fbe84c5a8b117c3bbeaaf5d",
                    "output_index": "0",
                    "amount": "16775501",
                    "is_our_address": false

it clearly says that it is not my address and that explains why it is not shown in `lncli walletbalance,` but how is that possible?

`lncli `getinfo` says "synced_to_chain" true and "synced_to_graph" true with correct "blockhight"

**Reply by guggero** [reply, marked as answer]
The funds are time locked (judging by the amount, probably for 2016 blocks). So you need to wait until the TX has that many confirmations. You can track the "maturity" of the time lock in `lncli pendingchannels`. The address does not appear as unspent or as "yours" in the transaction list because it is a (time lock) script address which isn't directly derivable from the seed, so `lnd` will sweep it into the wallet as soon as the time lock is over.

**Reply by ashjkl159** [reply]
hi understood, appreciate your explanation, thanks a lot
